app/layout/scenarios/sidebar/solutions/gap-analysis/component.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useCallback } from 'react';
+
+import { motion } from 'framer-motion';
+
+import Icon from 'components/icon';
+import Button from 'components/button';
+import List from 'layout/scenarios/sidebar/analysis/gap-analysis/list';
+import Toolbar from 'layout/scenarios/sidebar/analysis/gap-analysis/toolbar';
+
+// Icons
+import ARROW_RIGHT_SVG from 'svgs/ui/arrow-right-2.svg?sprite';
+import INFO_SVG from 'svgs/ui/info.svg?sprite';
+
+import { ScenariosSolutionsGapAnalysisProps } from './types';
+
+export const ScenariosSolutionsGapAnalysis: React.FC<ScenariosSolutionsGapAnalysisProps> = ({
+  onChangeSection,
+}: ScenariosSolutionsGapAnalysisProps) => {
+  const [search, setSearch] = useState(null);
+
+  const onSearch = useCallback((s) => {
+    setSearch(s);
+  }, []);
+
+  return (
+    <motion.div
+      key="details"
+      className="flex flex-col items-start justify-start min-h-0 overflow-hidden"
+      initial={{ opacity: 0 }}
+      animate={{ opacity: 1 }}
+      exit={{ opacity: 0 }}
+    >
+      <header className="flex items-center pt-5 pb-1">
+        <button
+          type="button"
+          className="flex items-center w-full space-x-2 text-left focus:outline-none"
+          onClick={() => {
+            onChangeSection(null);
+          }}
+        >
+          <Icon icon={ARROW_RIGHT_SVG} className="w-3 h-3 transform rotate-180 text-primary-500" />
+          <h4 className="text-xs uppercase font-heading">GAP ANALYSIS</h4>
+        </button>
+        <button
```

Comment:
Instead of using a custom info button, I created a Component called `InfoButton`. Could you change it?

Reply:
@mbarrenechea Done! However it doesn't look rounded for some reason 🤔 

![image](https://user-images.githubusercontent.com/545342/123086240-75c94280-d423-11eb-8e69-5e7b5dc91539.png)

I have checked how it's used in other parts of the app and I can't see any difference 🤷‍♂️ 
https://github.com/Vizzuality/marxan-cloud/pull/271/commits/22b7ba548a595bc3b44796024dd903ea61d7cda8#diff-4555a055e9c56541a441e74c31637073573d86d8186252e85bb3ea518b6de857R44

Reply:
Ok, add the class `flex-shrink-0` to the InfoButton > button

Reply:
@mbarrenechea done ✅ 
Hopefully it's ready to be merged now 🤞